pkgs/applications/misc/bluetooth_battery/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, fetchFromGitHub, buildPythonPackage, pybluez }:
+
+buildPythonPackage rec {
```

Comment:
This does not look like a library to me, but rather a case for `buildPythonApplication`.

Reply:
true, must have missed that when copying the expression from another package